changes/samdanirani-1.diff

```diff
@@ -0,0 +1,30 @@
+# Removing specialized twitch culture internet words
+
+# remove "pog" adjacent words
+- POGCHAMP
+- POGGER
+- POGGERS
+
+# remove "thoom" adjacent words
+- THOOMIN
+- THOOMING
+- THOOMER
+
+# "mald"
+- MALD
+- MALDED
+- MALDING
+
+# "tryhard"
+- TRYHARD
+- TRYHARDING
+- TRYHARDS
+
+# Misc.
+- BOTTING
+- BITRATE
+- BITRATES
+- CHAD
+- COLLAB
+- COLLABS
+- WOOT
```

Comment:
these are all (probably) valid and shouldn't be removed
maybe we'd need to discuss tryhard and woot - but the rest seem quite valid